_activities/cwp.md

```diff
@@ -28,7 +28,7 @@ we are reaching out to other HEP experiments around the world to participate.
 
 ## Final Roadmap Paper
 
-The aim is to produced the final CWP by December 15, 2017.
+The aim is finish the CWP by December 15, 2017.
```

Comment:
to finish

Reply:
Copied verbatim from Graeme's suggestion! I was also a bit surprised but thought that Graeme was the real expert there! @graeme-a-stewart do you confirm the mistake? I'll fix it and revert it if you say no...

Reply:
Yeah, I made a mistake, sorry!